bool location_request
---
int8 current_location